web/ui/react-app/src/Navbar.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import { Link, LinkGetProps } from "@reach/router";
+import {
+  Collapse,
+  Navbar,
+  NavbarToggler,
+  Nav,
+  NavItem,
+  NavLink,
+  UncontrolledDropdown,
+  DropdownToggle,
+  DropdownMenu,
+  DropdownItem,
+} from 'reactstrap';
+
+const isActive = ({ isCurrent }: LinkGetProps) => {
+  return isCurrent ? { style: { color: 'rgba(255,255,255,.5)' } } : null;
+}
+
+const isActiveDropDown = ({ isCurrent }: LinkGetProps) => {
+  return isCurrent ? { style: { backgroundColor: '#343a40', color: '#fff' } } : null;
+}
+
+const ExactNavLink = (props: any) => (
+  <Link getProps={isActive} {...props} />
+);
+
+const DropdownNavLink = (props: any) => (
+  <Link getProps={isActiveDropDown} {...props} />
+);
+
+const Navigation = () => {
+  const [isOpen, setIsOpen] = useState(false);
+  const toggle = () => setIsOpen(!isOpen);
+  return (
+    <Navbar className="mb-3" color="dark" expand="md">  
+      <Link className="pt-0 navbar-brand" to="/react/graph">Prometheus</Link>
+      <NavbarToggler onClick={toggle}/>
+      <Collapse isOpen={isOpen} navbar style={{ justifyContent: 'space-between' }}>
+        <Nav className="ml-0" navbar>
+          <NavItem>
+            <NavLink tag={ExactNavLink} to="/react/alerts">Alerts</NavLink>
+          </NavItem>
+          <NavItem>
+            <NavLink tag={ExactNavLink} to="/react/graph">Graph</NavLink>
+          </NavItem>
+          <UncontrolledDropdown nav inNavbar>
+            <DropdownToggle nav caret>Status</DropdownToggle>
+            <DropdownMenu left={true}>
+              <DropdownItem tag={DropdownNavLink} to="/react/status">Runtime & Build Information</DropdownItem>
+              <DropdownItem tag={DropdownNavLink} to="/react/flags">Command-Line Flags</DropdownItem>
+              <DropdownItem tag={DropdownNavLink} to="/react/config">Configuration</DropdownItem>
+              <DropdownItem tag={DropdownNavLink} to="/react/rules">Rules</DropdownItem>
+              <DropdownItem tag={DropdownNavLink} to="/react/targets">Targets</DropdownItem>
+              <DropdownItem tag={DropdownNavLink} to="/react/service-discovery">Service Discovery</DropdownItem>
+            </DropdownMenu>
+          </UncontrolledDropdown>
+          <NavItem>
+            <NavLink href="https://prometheus.io/docs/prometheus/latest/getting_started/">Help</NavLink>
+          </NavItem>
+        </Nav>
+        <Link to="../../graph">Return to classic UI</Link>
```

Comment:
I think it's fine to just make this a normal `NavItem` like the other menu items and just call it `Classic UI`. It's going to go away at some point anyway :)

Reply:
Yep, ok